common/src/WooCommerce.php

```diff
@@ -17,13 +17,20 @@ public function __construct(BasePlugin $plugin) {
         add_action('woocommerce_checkout_update_order_meta', [$this, 'set_order_language']);
         add_action('woocommerce_product_options_sku', [$this, 'addGtinOption']);
         add_action('woocommerce_admin_process_product_object', [$this, 'saveGtinOption']);
-        register_activation_hook($this->plugin->getPluginFile(), [$this, 'activateSyncReviews']);
+        add_action('init', [$this, 'activateSyncReviews']);
         register_deactivation_hook($this->plugin->getPluginFile(), [$this, 'deactivateSyncReviews']);
         add_action($this->getReviewsHook(), [$this, 'syncReviews']);
         add_action('wp_ajax_' . $this->getManualSyncAction(), [$this, 'manualReviewSync']);
     }
 
     public function activateSyncReviews() {
+        if (
+            defined('DISABLE_WP_CRON')
+            && DISABLE_WP_CRON
+            && get_option($this->plugin->getOptionName('product_reviews'))
+        ) {
+            add_action('admin_notices', [$this, 'wpCronDisabledNotice']);
+        }
```

Comment:
DISABLE_WP_CRON is usually set when a cronjob is set to run wp-cron.php without WordPress' automatic trigger.  So I don't think you need this notice.

(A possible alternative would be to look at the time the sync was last triggered, and when it was set up, to show a warning if it hasn't run for, eg, 24 hours.)